cds/modules/previewer/views.py

```diff
@@ -50,6 +51,14 @@ def preview_depid(pid, record, template=None, **kwargs):
 
 def preview_recid_embed(pid, record, template=None, **kwargs):
     """Return embedded player for video file."""
+    invenio_app = current_app.extensions.get('invenio-app', None)
```

Comment:
I would change this to:
```suggestion
    invenio_app = current_app.extensions['invenio-app']
```
It must fail if not there. WDYT?

Reply:
It should yes, thanks!